src/hotel-management-app/src/app/restaurant/owner-dashboard/owner-dashboard.component.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<mat-toolbar color="primary">
+    <span>My Restaurants</span>
+</mat-toolbar>
+
+<div class="restaurant-cards">
+
+    <h1 *ngIf="!restaurants.length">No restaurants to display!</h1>
+    <mat-card *ngFor="let restaurant of restaurants" class="restaurant-card">
+        <img mat-card-image [src]="restaurant.image" alt="{{ restaurant.name }}">
+        <mat-card-header>
+            <mat-card-title>{{ restaurant.name }}</mat-card-title>
+            <mat-card-subtitle>{{ restaurant.cuisine }}</mat-card-subtitle>
+            <mat-card-subtitle>{{ restaurant.description }}</mat-card-subtitle>
+        </mat-card-header>
+        <mat-card-content>
+            <p>Menu:</p>
+            <ul>
+                <li *ngFor="let item of restaurant.menu">
+                    {{ item.name }} - {{ item.price}}
+                    <button mat-button color="accent" (click)="editMenuItem(restaurant.id, item.id)" class="editmenu-btn">Edit</button>
+                    <button mat-button color="accent" (click)="deleteMenuItem(restaurant.id, item.id)" class="deletemenu-btn">Delete</button>
+                </li>
+            </ul>
+            <mat-slide-toggle (click)="toggle()">Show more options</mat-slide-toggle><br>
+        </mat-card-content>
+        <div *ngIf="!showOptions">
+        <mat-card-actions>
+            <button mat-raised-button color="accent" (click)="addMenuItem(restaurant.id)" class="addmenu-btn">Add Menu Item</button>
```

Comment:
can we have a uniform color for the buttons

Reply:
I actually changed this in the latest branch , so when I merge it with the latest branch it will be changed.